exaudfclient/base/BUILD

```diff
@@ -175,21 +150,6 @@ sh_library(
     })
 )
 
-SLOW_WRAPPER_BINARY_PY2="""$(location //:wrapper_generator_bin) "$(location exaudfclient_bin)" "$(location exaudfclient)" "$(location exaudfclient.template.sh)" """
-FAST_BINARY_PY2="""cp "$(location exaudfclient_bin)" "$(location exaudfclient)" """
-CREATE_BINARY_PY2_SCRIPT=select({
-        "//:fast_binary": FAST_BINARY_PY2,
-        "//:slow_wrapper": SLOW_WRAPPER_BINARY_PY2,
-        "//conditions:default": FAST_BINARY_PY2
-    })
-genrule(
-    name = "exaudfclient",
-    cmd = CREATE_BINARY_PY2_SCRIPT,
-    outs = ["exaudfclient"],
-    srcs = [":exaudfclient_bin", "//:libexaudflib_complete.so", "exaudfclient.template.sh","//:wrapper_generator_bin"],
-    output_to_bindir = True
-)
-
 SLOW_WRAPPER_BINARY_PY3="""$(location //:wrapper_generator_bin) "$(location exaudfclient_py3_bin)" "$(location exaudfclient_py3)" "$(location exaudfclient.template.sh)" """
 FAST_BINARY_PY3="""cp "$(location exaudfclient_py3_bin)" "$(location exaudfclient_py3)" """
 CREATE_BINARY_PY3_SCRIPT=select({
```

Comment:
Now if we remove the `exaudfclient`, it will be strange if we keep `exaudfclient_py3` (why only keeping the python version in the name, and not the Java version, R version etc....
Maybe we should rename now  `exaudfclient_py3`  =>  `exaudfclient`

Reply:
And with that we probably also should rename the Bazel variable names,
FAST_BINARY_PY3 -> FAST_BINARY
CREATE_BINARY_PY3_SCRIPT -> CREATE_BINARY_SCRIPT
...

Reply:
I had the same thought, but I left it this way, because at some point we should simply produce one binary per language.